server/i18n/en.json

```diff
@@ -7942,11 +7942,11 @@
   },
   {
     "id": "ent.elasticsearch.test_config.license.error",
-    "translation": "Your license does not support Elasticsearch."
+    "translation": "Your license does not support Search."
```

Comment:
We may want to be more explicit here because we are making this string potentially confusing. We could use a more generic label such as "Indexed Search" and also list available implementations ElasticSearch and OpenSearch.